src/components/sbb-tag-group/sbb-tag-group.tsx

```diff
@@ -117,17 +125,24 @@ export class SbbTagGroup implements ComponentInterface {
     return Array.from(this._element.querySelectorAll('sbb-tag')) as HTMLSbbTagElement[];
   }
 
+  private _readSlottedElements(): void {
+    this._slottedElements = Array.from(this._element.children).filter(
+      (e): e is HTMLElement => e instanceof window.HTMLElement,
+    );
+  }
+
```

Comment:
Should it be a getter like `_tags`?
What if the slotted content changes?

Reply:
> What if the slotted content changes?

For content changes there should be the `this._readSlottedElements()` on line 156

Reply:
you're right, soz :/